horizon/gopal/client.py

```diff
@@ -0,0 +1,55 @@
+import asyncio
+from typing import Optional, Awaitable, Callable
+
+from loguru import logger
+from opal_client import OpalClient
+from opal_client.engine.runner import PolicyEngineRunner
+
+from horizon.config import sidecar_config
+from horizon.gopal.runner import GopalRunner
+
+
+class GOPALClient(OpalClient):
+
```

Comment:
not sure i understand your assumptions in this class.
if you run the go service, you need to wait for it to become ready, and that should reflect in the PDP health and readiness checks. where is this code?

Reply:
missing the reflection in the healthcheck, will add

Reply:
looks like you fixed it :)